File: C/IIIT/armstrong.c
#include <stdio.h>
#include <math.h>
void main(){
    int num,sum = 0;
    printf("enter a three digit number : \n");
    scanf("%d", &num);
    int a = num;
    while(num){
        int digit = num % 10;
        sum += pow(digit,3);
        num /= 10;
    }
    if(sum == a)
        printf("its an armstrong number");
    else
        printf("it isn't an armstrong number");
}

File: python files/IIIT/python/lab6/armstrong.py
a = int(input("Enter a number : \n"))
b = a
sum = 0

while b>0:
    digit = b % 10
    sum = sum + digit**3
    b = b // 10

if sum == a:
    print("It's an armstrong number")
else:
    print("It isn't an armstrong number")